elfpy/pricing_models/yieldspace.py

```diff
@@ -396,23 +385,16 @@ def calc_in_given_out(
                 )
                 * share_price  # convert to base
             )
-            # The fees are calculated as the difference between the bonds
-            # received and the base paid without slippage times the fee
-            # percentage. This can also be expressed as:
-            #
-            # fee = (1 - p) * phi * d_y
-            fee = (1 - spot_price) * trade_fee_percent * d_bonds
-            logging.debug(
-                ("fee = (1 - spot_price) * _fee_percent * d_bonds = (1 - %g) * %g * %g = %g"),
-                spot_price,
-                trade_fee_percent,
-                d_bonds,
-                fee,
+            curve_fee, gov_curve_fee = self.calc_curve_fee_split(
```

Comment:
let's keep the fees calcs here inline and just add the gov fee part

Reply:
done in ebe871de8cd655cc7403dfa24ee00e25b9eb2fe7

Reply:
removed in ebe871de8cd655cc7403dfa24ee00e25b9eb2fe7